src/validations/comment.js

```diff
@@ -1,14 +1,20 @@
 import { check } from 'express-validator/check';
 
-const uuidRegularExpression = new RegExp(/^[0-9a-f]{8}-[0-9a-f]{4}-[1-5][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}$/i);
+const uuidRegularExpression = new RegExp(
+  /^[0-9a-f]{8}-[0-9a-f]{4}-[1-5][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}$/i
```

Comment:
Nice work, but you make use of is UUID from express validation instead of using the above regex so we can have a uniform validation schema.

Reply:
done